Mobile-Buy-SDK.podspec

```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |s|
 
   s.name                = 'Mobile-Buy-SDK'
-  s.version             = '4.0.0'
+  s.version             = '5.0.0'
```

Comment:
This should probably be `4.0.1` as it received no feature updates. The version also needs to be updated in the Xcode project in 4 separate targets.

Reply:
There was a breaking change in the API and hence the mobile SDK is also bumped up with a major version number